### ISSUE TITLE
No fibers found...

### ISSUE THREAD

**Issue by acinader** [scriby/asyncblock#29]
in 2.2.4, I'm getting the following error:

```
[dtsgen]    Error: Cannot find module 'fibers'
[dtsgen]        at Function.Module._resolveFilename (module.js:336:15)
[dtsgen]        at Function.Module._load (module.js:278:25)
[dtsgen]        at Module.require (module.js:365:17)
[dtsgen]        at require (module.js:384:17)
[dtsgen]        at /Users/arthur/code/shopstyle-node-boilerplate/node_modules/asyncblock/lib/asyncblock.js:13:23
[dtsgen]        at handleFiber (/Users/arthur/code/shopstyle-node-boilerplate/node_modules/asyncblock/lib/asyncblock.js:95:21)
[dtsgen]        at asyncblock (/Users/arthur/code/shopstyle-node-boilerplate/node_modules/asyncblock/lib/asyncblock.js:28:16)
[dtsgen]        at module.exports (/Users/arthur/code/shopstyle-node-boilerplate/node_modules/asyncblock/lib/asyncblock.js:44:5)
[dtsgen]        at processGenerate (/source/cli.ts:36:3)
[dtsgen]        at Object.<anonymous> (/source/cli.ts:31:3)
```

i'm no expert, so if this doesn't look like you're issue, sorry.

I see you're last commit changed from fibers to fibers-scriby, but appears that something is still looking for fibers?

**Comment by scriby** [comment]
Sorry, mistake on my part, I'll get it fixed up shortly.

**Comment by scriby** [comment]
Should be fixed in 2.25. Let me know if it's not.
